app/src/pages/inside/common/utils.js

```diff
@@ -135,3 +139,38 @@ export const createStepActionDescriptors = (params) => {
     },
   ];
 };
+
+export const getGroupedDefectTypesOptions = (
+  defectTypes,
+  formatMessage,
+  defectTypesSequence = DEFECT_TYPES_SEQUENCE,
+) => {
+  let defectTypesOptions = [];
+  defectTypesSequence.forEach((defectTypeId) => {
+    const defectTypeGroup = defectTypes[defectTypeId];
+    defectTypesOptions.push({
+      label: formatMessage(defectTypesLocalization[`${defectTypeGroup[0].typeRef}_TOTAL`]),
+      value: `${DEFECT_STATISTICS_BASE}${defectTypeGroup[0].typeRef.toLowerCase()}$total`,
+      groupId: defectTypeGroup[0].typeRef,
+      color: defectTypeGroup[0].color,
+      typeRef: defectTypeGroup[0].typeRef,
+    });
+    defectTypesOptions = defectTypesOptions.concat(
+      defectTypeGroup.map((defectType) => ({
+        groupRef: defectType.typeRef,
+        value: `${DEFECT_STATISTICS_BASE}${defectType.typeRef.toLowerCase()}$${defectType.locator}`,
+        label: defectTypesLocalization[defectType.locator]
+          ? formatMessage(defectTypesLocalization[`${defectType.typeRef}`])
```

Comment:
Should we update condition accordingly? Seems like now it will return false every time.
```javascript
defectTypesLocalization[defectType.typeRef]
? formatMessage(defectTypesLocalization[defectType.typeRef])
: defectType.longName
```

Reply:
Sure